src/Sylius/Behat/Page/Admin/Taxon/CreatePage.php

```diff
@@ -121,6 +121,30 @@ public function attachImageWithCode($code, $path)
     }
 
     /**
+     * {@inheritDoc}
+     */
+    public function moveUp(TaxonInterface $taxon)
+    {
+        $this->moveLeaf($taxon, self::MOVE_DIRECTION_UP);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function moveDown(TaxonInterface $taxon)
+    {
+        $this->moveLeaf($taxon, self::MOVE_DIRECTION_DOWN);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getFirstLeafName(TaxonInterface $parentTaxon = null)
```

Comment:
This method is quite fragile. Refactoring it to `getChildrenOfTaxon(TaxonInterface $parentTaxon = null)` would allow for more flexibility.

```php
return array_map(function (Something $element) { 
  return $element->getText(); 
}, $this->getLeafs($parentTaxon));
```

Then asserting for having some taxons under given one or the last one wouldn't require us to create yet another method.